oak-jcr/src/test/java/org/apache/jackrabbit/oak/jcr/delegate/SessionDelegateTest.java

```diff
@@ -68,4 +74,38 @@ public void testRefreshUnaware() {
         delegate.refresh(false);
         verify(pp, times(6)).refresh();
     }
+    
+    @Test(expected = RepositoryException.class)
```

Comment:
Shouldn't these tests now expect a `ConstraintViolationException`?

Reply:
The contract states just a ```RepositoryException```, so it makes sense just to test for that.
Although one could argue, that a change in that code, which also throws a ```RepositoryException``` (or an exception inheriting from it) would be caught as well.

As this is the test for this implementation,I will change it to the ```ConstraintViolationException```.